### DISCUSSION TITLE
feat(misconf) junit.tpl misses file reference

### DISCUSSION THREAD

**Discussion by atombrella** [aquasecurity/trivy#7062] [Ideas]
### Description

We are using trivy for scanning Helm charts in Azure DevOps. The results are rendered using the junit.tpl However, it seems that there's a file attribute in the XML specification that may (I don't know how Azure will treat it) improve navigation.

It's probably just `file="{{ .Target }}"` that misses from the template.

https://github.com/testmoapp/junitxml

### Target

None

### Scanner

Misconfiguration

**Comment by simar7** [comment]
hi @atombrella thanks for the insight, we would welcome your PR if you decided to improve the template!
